gradle/java/folder-layout.gradle

```
@@ -25,13 +25,11 @@ allprojects {
       test.resources.srcDirs = ['src/test-files']
     }
 
-    task copyTestResources(type: Copy) {
+    processTestResources {
```

let's use this to avoid eager task configuration:

```suggestion
    tasks.named('processTestResources') {
```